docs/vhlo.md

```diff
@@ -0,0 +1,128 @@
+# The VHLO Dialect
+
+## What is the VHLO Dialect?
+
+The VHLO (Versioned StableHLO) Dialect is used for serialization and stability.
+It provides a snapshot of the StableHLO dialect at a given point in time,
+denoted by version number.
+
+VHLO is an **add-only dialect** with **versioned ops**, which means that once an
```

Comment:
"versioned ops, types and attributes"? If that makes sense, let's also drop "the ops" in "the ops semantics" at the end of this sentence as well as reformulate the next paragraph to not focus just on ops.